cg/apps/demultiplex/sample_sheet/sample_sheet_validator.py

```diff
@@ -156,82 +142,45 @@ def _validate_override_cycles(self) -> None:
+        LOG.info("Samplesheet passed validation")
 
-    def _validate_bcl2fastq(self):
-        """Determine if the Bcl2Fastq sample sheet is valid, which means:
-        - Sample sheet is Bcl2Fastq
-        - The samples have the correct attributes
-        """
-        self._validate_sample_sheet_is_correct_type(bcl_converter=BclConverter.BCL2FASTQ)
-        self._validate_samples(sample_type=FlowCellSampleBcl2Fastq)
-
-    def validate_sample_sheet_from_content(
-        self, content: list[list[str]], bcl_convert: str
-    ) -> None:
-        """
-        Call the proper validation depending on the sample sheet type or the given bcl converter.
-        Raises:
-            SampleSheetError: If the sample sheet is not valid.
-        """
-        self.set_sample_sheet_content(content)
-        if bcl_convert == BclConverter.BCLCONVERT:
-            LOG.debug("Validating BCLConvert sample sheet")
-            self._validate_bcl_convert()
-            LOG.info("Samplesheet passed BCLConvert validation")
-        elif bcl_convert == BclConverter.BCL2FASTQ:
-            LOG.debug("Validating Bcl2fastq sample sheet")
-            self._validate_samples(sample_type=FlowCellSampleBcl2Fastq)
-            LOG.info("Samplesheet passed Bcl2Fastq validation")
-        else:
-            LOG.error("Could not determine sample sheet type")
-            raise SampleSheetError("Could not determine sample sheet type")
-
-    def validate_sample_sheet_from_file(self, file_path: Path, bcl_converter: str) -> None:
+    def validate_sample_sheet_from_file(self, file_path: Path) -> None:
         """
         Validate a sample sheet given the path to the file.
         Raises:
             SampleSheetError: If the sample sheet is not valid.
         """
-        self.validate_sample_sheet_from_content(
-            content=ReadFile.get_content_from_file(file_format=FileFormat.CSV, file_path=file_path),
-            bcl_convert=bcl_converter,
+        content: list[list[str]] = ReadFile.get_content_from_file(
+            file_format=FileFormat.CSV, file_path=file_path
         )
+        self.validate_sample_sheet_from_content(content)
 
-    def get_sample_sheet_object_from_file(self, file_path: Path, bcl_converter: str) -> SampleSheet:
+    def get_sample_sheet_object_from_file(self, file_path: Path) -> SampleSheet:
         """Return a sample sheet object given the path to the file.
         Raises:
             SampleSheetError: If the sample sheet is not valid.
         """
-        sample_type: Type[FlowCellSample] = BCL_CONVERTER_TO_FLOW_CELL_SAMPLE[bcl_converter]
-        self.validate_sample_sheet_from_file(file_path=file_path, bcl_converter=bcl_converter)
+        self.validate_sample_sheet_from_file(file_path)
         samples: list[FlowCellSample] = get_flow_cell_samples_from_content(
             sample_sheet_content=self.content,
-            sample_type=sample_type,
+            sample_type=FlowCellSampleBCLConvert,
```

Comment:
I think we are only left with a single sample type now, will you refactor this in another pr?

Reply:
yes!